pom.xml

```diff
@@ -177,6 +169,13 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>31.0.1-jre</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Do you really need it?

Reply:
We removed it from the activepivot jar and we use it in the tests in mac